tools/editor/plugins/script_editor_plugin.cpp

```diff
@@ -1256,6 +1256,12 @@ void ScriptEditor::_menu_option(int p_option) {
 				{
 					int begin = tx->get_selection_from_line();
 					int end = tx->get_selection_to_line();
+					int column_end = tx->get_selection_to_column();
+
+					// End of selection ends on the first column of the last line, ignore it.
+					if(column_end == 0)
```

Comment:
You could use directly ``if(tx->get_selection_to_column() == 0)`` to avoid having to initialize a variable to use it once.